mantidimaging/gui/widgets/mi_mini_image_view/view.py

```diff
@@ -166,3 +170,8 @@ def update_sibling_histograms(self):
         for img_view in self.histogram_siblings:
             with BlockQtSignals(img_view.hist):
                 img_view.hist.setLevels(*hist_range)
+
+    def use_brightness_percentiles(self, image: np.ndarray, percent_low: int, percent_high: int, *args, **kwargs):
```

Comment:
We don't need the `*args, **kwargs` part here, as its not being passed through in the this method.